lib/searchable_by/column.rb

```diff
@@ -58,8 +58,8 @@ def str_condition(scope, value)
     end
 
     def escape_term!(term)
```

Comment:
Comment re CodeQL warnings for "next time", doens't appear to be any concise mechanism for inline alert suppression

Reply:
Dismissed in the UI